pages/place/list/views/index.jsx

```diff
@@ -59,6 +59,14 @@ export default function Places() {
   schema.nacwos.label = <Fragment><Acronym>NACWO</Acronym>s</Fragment>;
   schema.nvssqps.label = <Fragment><Acronym>NVS</Acronym>s / <Acronym>SQP</Acronym>s</Fragment>;
 
+  const actions = <Fragment>
+    <Link page="place.list" query={{csv: 1}} label={<Snippet>actions.download</Snippet>} />
```

Comment:
One thing to note here is that it will remove any filters that have been applied and always return the full set.

I'm not sure if this is intentional or not.

Reply:
I believe that's what was intended - isn't that the case with all of our CSV downloads (i.e. return the full set)? It may not be obvious from the UI what to expect however.

Reply:
Confirmed with Ben - link text now says "Download all as CSV" rather than just "Download as CSV"

Reply:
For the record, the other places we have CSV downloads (project content search, perf metrics) respect whatever filters have been applied.